### PR TITLE
New docs theme

### PR DESCRIPTION
**Pull request by coretl**

### CODE REVIEW

**Comment by coretl** [comment]
https://diamondlightsource.github.io/python3-pip-skeleton-cli/dev/index.html

File: .github/CONTRIBUTING.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+Contributions and issues are most welcome! All issues and pull requests are
+handled through GitHub_. Also, please check for any existing issues before
+filing a new one. If you have a great idea but it involves big changes, please
+file a ticket before making a pull request! We want to make sure you don't spend
+your time coding something that might not fit the scope of the project.
+
+.. _GitHub: https://github.com/DiamondLightSource/python3-pip-skeleton-cli/issues
+
+Issue or Discussion?
```
Comment (gilesknap) [inline .github/CONTRIBUTING.rst:12]: Discussions is a nice idea. But we need to turn it on in the repo. 

We need a guide to configuring github repo and it should include this.

File: .github/workflows/code.yml [hunk 2]
```diff
@@ -171,8 +171,7 @@ jobs:
         uses: softprops/action-gh-release@1e07f4398721186383de40550babbdf2b84acfc5 # v0.1.14
         with:
           prerelease: ${{ contains(github.ref_name, 'a') || contains(github.ref_name, 'b') || contains(github.ref_name, 'rc') }}
-          files: |
-            dist/*
+          files: dist/*
```
Comment (gilesknap) [inline .github/workflows/code.yml:174]: I assume you have verified that this works? I thought I had to add the /* to make it work previously - but perhaps we had subfolders then?

File: .github/workflows/docs_clean.yml [hunk 3]
```diff
@@ -32,10 +32,9 @@ jobs:
 
       - name: update index and push changes
         run: |
-          echo removing redundant documentation version ${{ env.remove_me }}
           rm -r ${{ env.remove_me }}
-          sed -i /${{ env.remove_me }}/d versions.txt
+          python make_switcher.py --remove ${{ env.remove_me }} ${{ github.repository }} switcher.json
```
Comment (gilesknap) [inline .github/workflows/docs_clean.yml:36]: make switcher is neat

File: docs/user/how-to/run-container.rst [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+Pre-built containers with python3-pip-skeleton-cli and its dependencies already
+installed are available on `Github Container Registry
+<https://ghcr.io/DiamondLightSource/python3-pip-skeleton-cli>`_.
+
+Starting the container
+----------------------
+
+To pull the container from github container registry and run::
+
+    $ docker run --rm ghcr.io/DiamondLightSource/python3-pip-skeleton-cli:main --version
```
Comment (gilesknap) [inline docs/user/how-to/run-container.rst:13]: I thought I was suggesting a change but looks like I committed it.

Comment (gilesknap) [reply docs/user/how-to/run-container.rst:13]: (added --rm)

**Review by gilesknap — COMMENTED** [review]
I'm just going to do a pull of the latest into mciwb and verify that it looks good, then will approve.

File: .github/workflows/code.yml [hunk 5]
```diff
@@ -139,7 +139,7 @@ jobs:
         uses: actions/upload-artifact@v3
         with:
           name: dist
-          path: dist/*
+          path: dist
```
Comment (gilesknap) [inline .github/workflows/code.yml:142]: I assume this is working? I had to add /* earlier but maybe that was when we had subfolders?

**Review by gilesknap — COMMENTED** [review]
The import into mciwb was all good. The API docs look particularly nice https://gilesknap.github.io/mciwb/containers/user/reference/api.html.

I had to do merge conflicts over python-pip-skeleton-->mciwb and I'm not sure why.

Now taking a look at why the build failed.

**Comment by gilesknap** [comment]
CI Now fixed.

**Review by gilesknap — APPROVED** [review]
All good

File: .github/workflows/code.yml (hunk 5 above)
Comment (coretl) [reply .github/workflows/code.yml:142]: Seems to be working, There is a single artifact called `dist` that contains the wheel, sdist and `requirements*.txt`:
https://github.com/DiamondLightSource/python3-pip-skeleton-cli/suites/8261099796/artifacts/361392560

**Pull request merged**